modules/source-bookshelf/lib/base_methods.js

```diff
@@ -29,16 +31,28 @@ exports.addFilter = function (source) {
 };
 
 exports.addCreate = function (source) {
-  source.model.create = function (params) {
+  source.model.create = function (params, toManyRels) {
     return this.forge(params).save(null, {method: 'insert'}).then(function (model) {
-      return this.forge({id:model.id}).fetch();
+      var self = this;
+      return bluebird.each(toManyRels, function(rel) {
```

Comment:
do we care about the order of operations for this? if no, what about using `bluebird.all`?